src/app/meta-campaign/meta-campaign.component.html

```diff
@@ -1,48 +1,94 @@
+    [description]="
+      fund && fund.description ? fund.description : campaign.summary
+    "
     (heroSearch)="onMetacampaignSearch($event)"
   ></app-hero>
 
-  <div class="b-container" *ngIf="campaign && children">
-    <app-filters
-      *ngIf="campaign.campaignCount && campaign.campaignCount > 1"
-      [hasTerm]="hasTerm"
-      [selectedSort]="selectedSort"
-      [reset]="resetSubject.asObservable()"
-      (filterApplied)="onFilterApplied($event)"
-      (sortApplied)="onSortApplied($event)"
-    ></app-filters>
+  <div *ngIf="campaign && children">
+    <div class="b-container">
+      <app-filters
+        *ngIf="campaign.campaignCount && campaign.campaignCount > 1"
+        [hasTerm]="hasTerm"
+        [isHomepage]="false"
+        [selectedSort]="selectedSort"
+        [selectedFilters]="query"
+        [reset]="resetSubject.asObservable()"
+        (filterApplied)="onFilterApplied($event)"
+        (sortApplied)="onSortApplied($event)"
+        (numberOfCardsApplied)="onNumberOfCardsApplied($event)"
+        (clearFiltersApplied)="onClearFiltersApplied()"
+      ></app-filters>
 
-    <div *ngIf="campaign && children.length === 0 && !loading && !filterError">
-      <p class="error" aria-live="polite">
-        We can't find any campaigns matching this search but there are lots more to choose from.
-        <a (click)="setDefaultFilters()">View all participating campaigns here</a>.
-      </p>
-    </div>
+      <div
+        *ngIf="campaign && children.length === 0 && !loading && !filterError"
+      >
+        <p class="error" aria-live="polite">
+          We can't find any campaigns matching this search but there are lots
+          more to choose from.
+          <a (click)="setDefaultFilters(true)"
+            >View all participating campaigns here</a
+          >.
+        </p>
+      </div>
 
-    <div class="c-grid">
-      <div *ngFor="let campaign of children">
-        <div
-          infiniteScroll
-          (scrolled)="onScroll()"
-          [infiniteScrollDistance]="20"
-          [scrollWindow]="true"
-          [alwaysCallback]="true"
-        ></div>
-        <app-campaign-card [campaign]="campaign" [inFundContext]="!!fund"></app-campaign-card>
+      <div class="c-grid">
+        <div *ngFor="let campaign of children">
+          <app-campaign-card
+            [campaign]="campaign"
+            [inFundContext]="!!fund"
+          ></app-campaign-card>
+        </div>
       </div>
-    </div>
 
-    <mat-spinner *ngIf="loading" color="primary" diameter="30"></mat-spinner>
-  </div>
+      <div class="align-center padding-top-s">
+        <button
+          class="b-db b-light align-center"
+          mat-raised-button
+          color="primary"
+          type="submit"
+          (click)="loadMore()"
+        >
+          Load More
+        </button>
+      </div>
+
+      <div *ngIf="!hasMore" class="t-align-center padding-top-s" >
```

Comment:
Why do we need this? And why are we removing infinite scroll for the meta-campaign page?